CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Here is a template for new release sections
 - Changed and added pytests in for `C2` (#520)
 - All energyProviders that have key `FILENAME` (and, therefore, a timeseries), are now of `DISPATCHABILITY = False`(#520)
 - Changed structure of `E2.lcoe_assets()` so that each asset has a defined LCOE_ASSET. If `sum(FLOW)==0` of an asset, the LCOE_ASSET (utilization LCOE) is defined to be 0 (#520)
+- Color lists for plots are provided by user and are not hard coded anymore (#527)
```

Comment:
How does the user choose a set of colors - isn´t it the developer?

Reply:
true, that it is still the developer, however the previous implementation was that the colors where arbitrarily hardcoded in the function. If you had more assets than color there would have been ignored and not being plotted. Now, I made so that we can provide a list of color to the function (if we want to) of it takes a color from the default color with periodic boundary conditions (when index is larger than the color list we take the modulo with respect to the color list's length)